relational/project.py

```diff
@@ -86,6 +86,10 @@ class Project:
     """A way to abstract some of the details of handling multiple records for a
     project, from multiple sources."""
 
+    def __str__(self):
+        return '%s => { roots: %s, children %s }' % (self.id, self.roots,
+                                                     self.children)
```

Comment:
A general style nit: args should either all fit on one line or be one-per-line. For example, I'd format this like:

```python
def __str__(self):
    return '%s => { roots: %s, children %s }' % (
        self.id, self.roots, self.children)
```

Reply:
ooh, thanks, i think i have a lot of  places to fix like this